Resources/Prototypes/Entities/Objects/Weapons/Throwable/grenades.yml

```diff
@@ -45,6 +45,11 @@
     intensitySlope: 3
     totalIntensity: 120 # about a ~4 tile radius
     canCreateVacuum: false
+  - type: OnUseTimerTrigger
+    beepSound:
+      path: "/Audio/Effects/beep1_double.ogg"
+    initialBeepDelay: 0
+    beepInterval: 666 # single beep at start
```

Comment:
cant you change this instead of duplicating the sound file?

Reply:
No, you can't set amount of beeps

Reply:
i mean the second beep doesnt have to be right after, the timer is 3.5 seconds so setting this to 2 (or 1.5 for a third beep) should do the trick right?

Reply:
the idea was to indicate the arming of grenade but not give away its location

Reply:
Could this just be nullable instead instead of the high value?

Reply:
using high value doesn't require code changes and special treatment so its better imo

Reply:
it has a comment explaining so i also think its fine

Reply:
just keep it to one sound file and increase the volume a bit with the beepSound sound params